lom/src/plugins/sonic/plugin_integration_tests/detection/link/linkcrc/main.go

```diff
@@ -0,0 +1,159 @@
+package main
+
+import (
+        "os/exec"
+        "strings"
+        "fmt"
+        "github.com/go-redis/redis"
+        "lom/src/plugins/sonic/plugin_integration_tests/utils"
+        "lom/src/lib/lomcommon"
+        "lom/src/lib/lomipc"
+        "strconv"
+        "time"
+        "os"
+        "io/ioutil"
+        "lom/src/plugins/sonic/plugin/detection/link/linkcrc"
+        "lom/src/plugins/plugins_common"
+        "context"
+)
+
+const (
+        redis_address                    = "localhost:6379"
```

Comment:
Add an ADO that we get this address from global rc file. This ADO can be in back log.

Reply:
I have created this ado task https://msazure.visualstudio.com/One/_workitems/edit/24130880